operational_analysis/methods/plant_analysis.py

```diff
@@ -831,6 +832,11 @@ def run_AEP_monte_carlo(self):
                 inputs = inputs.reshape(-1,1)
             gross_lt = fitted_model.predict(inputs)
 
+            # Annual values of lt gross energy, needed for IAV
+            reg_inputs_lt['gross_lt'] = gross_lt
+            gross_lt_annual = reg_inputs_lt['gross_lt'].resample('12MS').sum().values
```

Comment:
minor detail, but if the data has monthly time resolution I would think the normalization to 30-day months should be undone before calculating `gross_lt_annual`

Reply:
Nice catch. Yes, I agree with you.